core/rpc-servers/src/lib.rs

```diff
@@ -60,7 +60,11 @@ pub fn rpc_handler<M: PubSubMetadata>(
 mod inner {
 	use super::*;
 
+	/// Type synonym for the JSONRPC HTTP `Server` data structure from the
+	/// `jsonrpc-http-server` crate.
```

Comment:
I don't know what to think about this doc, I think this can be removed

Reply:
Then [this](https://github.com/paritytech/substrate/pull/3886/files#diff-435a815e40d1829b9fd17fb03813d2edR19) will fail. I know it's not the most informative comment, but it's probably better than nothing, and excusing these two `types` also doesn't seem right.